libqtile/backend/x11/window.py

```diff
@@ -1164,16 +1164,14 @@ def change_layer(self, up=True, top_bottom=False):
                 higher.sort(key=lambda wid: stack.index(wid))
 
             for wid in [w for w in lower if stack.index(w) > index]:
-                self.qtile.windows_map[wid].window.configure(
-                    stackmode=xcffib.xproto.StackMode.Below, sibling=same[0]
-                )
+                win = self.qtile.windows_map.get(wid, self.qtile.core.override_redirect_map[wid])
```

Comment:
Looks up the wid in windows_map and, if it's not there then it must be in the override_redirect map.